plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcDynamicFilteringConfig.java

```diff
@@ -23,9 +23,9 @@
 public class JdbcDynamicFilteringConfig
 {
     private boolean dynamicFilteringEnabled = true;
-    // 20s allows DF collection from dimensional tables as well as
-    // some larger build side subqueries
-    private Duration dynamicFilteringWaitTimeout = new Duration(20, SECONDS);
+    private Duration dynamicFilteringWaitTimeout = new Duration(1, SECONDS);
+    // dynamic filter timeout for filters we cannot estimate build side size or connector does not present table statistics
+    private Duration unestimatableDynamicFilteringWaitTimeout = new Duration(20, SECONDS);
```

Comment:
New set of configs is quite difficult to grasp for users.
How about we simplify this in the following way:
Keep config here same as before.
When there are estimates, the Determine... rule decides DF timeout.
When there are no estimates, connector uses existing `dynamicFilteringWaitTimeout` config.
We can apply this policy consistently in all connectors instead of using "max" of the two timeouts.

Reply:
> we want still await at least 1s (dynamicFilteringWaitTimeout session property) time for DF to prevent to fire query too soon if our estimate is not exact

I expect that we've tuned the estimate thresholds to a point where they can be trusted. If we're not able to trust estimates in certain situations, we should bail out or tune limits lower. We shouldn't need to hedge against the case where something is estimated.
Still, if there is no other way around it, then I would rather have that `1s` wait be a hard-coded constant than yet another config.